### PR TITLE
download button final

### PR DESCRIPTION
**Pull request by saikiranreddy098**
Download button is added in Java script
corresponding routes are in apps/citadel/document_management/routes
and handler for these routes are in services/document_management_service.py

### CODE REVIEW

File: apps/citadel/company_management/routes.py
Comment (sidhantsambyal) [inline apps/citadel/company_management/routes.py:1]: No need to add those files in Pull request with no changes. Changes of spaces or up down imports

Comment (saikiranreddy098) [reply apps/citadel/company_management/routes.py:1]: cause this is a format followed by viresh sir seperating python specific library imports from actual application specific imports by giving a blank space in between.

**Review by sidhantsambyal — APPROVED** [review]
conflicts resolved and checked

**Review by RupalyThakur — APPROVED** [review]
As conflicts are checked and resolved

**Pull request merged**